lib/pages/settings_dashboard/settings/settings.dart

```diff
@@ -223,8 +223,13 @@ class SettingsController extends State<Settings> with ConnectPageMixin {
   @override
   void dispose() {
     onAccountDataSubscription?.cancel();
-    avatarUriNotifier.dispose();
-    displayNameNotifier.dispose();
+    if (avatarUriNotifier.value != null) {
+      avatarUriNotifier.dispose();
+    }
+
+    if (displayNameNotifier.value != null) {
```

Comment:
why we need check value first?

Reply:
If you click on the Settings tab and immediately switch to another tab, there will be an error when `dispose `. To be sure, I will check to see if the variable already has a value and then dispose because the value is initialized after init.